borg/helpers.py

```diff
@@ -235,16 +235,21 @@ def parse_timestamp(timestamp):
         return datetime.strptime(timestamp, '%Y-%m-%dT%H:%M:%S').replace(tzinfo=timezone.utc)
 
 
+def load_excludes(fh):
```

Comment:
> maybe rather use f or file or exclude_file instead of fh.

“file” is a builtin and [should be avoided](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles) or suffixed.

> fh is a very popular variable name meaning file handle in the C / OS sense (that is NOT a python file object, but an integer

“fd” is common for integer file descriptors, “fh” is often used for file objects (“file handle”).